src/huggingface_hub/repocard_data.py

```diff
@@ -76,6 +81,38 @@ class EvalResult:
 
 @dataclass
 class CardData:
+    def __init__(self, **kwargs):
```

Comment:
same here, should we make this abstract

Reply:
Yea...this one bothers me a bit more. Mostly because in `modelcards` this was the name of the object you'd use, so it could cause some confusion.